src/spikeinterface/generation/tests/test_template_fetch.py

```diff
@@ -1,13 +1,51 @@
-import pytest
```

Comment:
Could we change the name to test_template_database.py

Reply:
I think this makes sense. Yes.

Reply:
Done.